tests/telemetry/test_telemetry.py

```diff
@@ -133,3 +135,54 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, localhost):
     result = str(show_gnmi_out)
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
+
+def test_sysuptime(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """
+    @summary: Run pyclient from ptfdocker and test the dataset 'system uptime' to check
+              whether the value of 'system uptime' was an integer and whether the value was
+              updated correctly.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    docker_present = verify_telemetry_dockerimage(duthost)
+    if not docker_present:
+        pytest.skip("docker-sonic-telemetry is not part of the image")
+
+    logger.info("start test the dataset 'system uptime'")
+    setup_telemetry_forpyclient(duthost)
+
+    # Wait until telemetry was restarted
+    pytest_assert(wait_until(100, 10, duthost.is_service_fully_started, "telemetry"), "TELEMETRY not started.")
+    logger.info("telemetry process restarted. Now run pyclient on ptfdocker")
+
+    # Wait until the TCP port was opened
+    dut_ip = duthost.mgmt_ip
+    wait_tcp_connection(localhost, dut_ip, TELEMETRY_PORT, timeout_s=60)
```

Comment:
Can you make the setup of telemetry on DUT a fixture as well to avoid duplication? It looks like more than one test cases need this setup. This fixture does not need to be autouse. Test cases can explicitly specify the dependency of the setup fixture.

Reply:
Yes. I will do that.

Reply:
Updated! Please have a review.